### PR TITLE
Disable WLCS tests on LP/arm* (as they are currently flaky)

### PR DESCRIPTION
**Pull request by AlanGriffiths**
Workaround: #2474

### CODE REVIEW

**Review by wmww — APPROVED** [review]
I think it's a good idea, but not sure if I'm the one to approve it
bors delegate+

**Comment by RAOF** [comment]
We'll experiment with trying qemu from 22.04 and see if that makes things more stable before landing this.

**Comment by Saviq** [comment]
#2490

File: debian/rules [hunk 1]
```diff
@@ -41,6 +41,11 @@ endif
 # control, so disable it on non-Ubuntu builds
 ifeq ($(filter Ubuntu,$(DEB_VENDOR)),)
 	COMMON_CONFIGURE_OPTIONS += -DMIR_RUN_WLCS_TESTS=OFF
+else
+  # Disable wlcs tests on Launchpad armhf, arm64 (MirServer/mir#2474)
+  ifneq ($(filter armhf arm64,$(DEB_HOST_ARCH)),)
+    COMMON_CONFIGURE_OPTIONS += -DMIR_RUN_WLCS_TESTS=OFF
+  endif
```
Comment (Saviq) [inline debian/rules:48]: If at all, we should only disable them when cross-building - which is where we see those failures.

But let's see where does #2490 take us.

**Pull request closed without merging**